### DISCUSSION TITLE
Roughly enough items not showing as a mod in mod menu and not working in game

### DISCUSSION THREAD

**Discussion by absurdistMaboroshi** [shedaniel/RoughlyEnoughItems#1165] [Q & A] [answered]
![image](https://user-images.githubusercontent.com/116382456/197261053-adc7b9d3-0e80-4876-b21f-de2e44e5d6f0.png)
![image](https://user-images.githubusercontent.com/116382456/197261100-4e3c8683-7521-49d5-9d71-225430753d1b.png)
I have made sure im running fabric everything and fabric is up to date. unsure why this is happening

**Comment by dpaulos6** [comment]
**Why hasn't this been answered yet?** 
The problem persists in **2023**. I'm fully up to date, running multiple mods without a problem, but REI doesn't work just as OP mentions. Could someone please reply to this, my last option is switching to JEI.

I don't like switching mods, I prefer helping developers fix their mods than switching because I found an issue. I'm also a dev and understand the frustration behind finding and solving bugs, but this has been **up for a year** and there has been no answer yet, I'm just making sure no one else needs to search countless sources to find a solution and not find one.

**Reply by shedaniel** [reply, marked as answer]
You likely installed the Forge version of REI and not noticing it

**Reply by dpaulos6** [reply]
Well, you are correct, probably a miss-click. Since they have exact names I couldn't tell.

I suggest adding which mod loader they use on the file name to prevent this from happening.
If they were like **RoughlyEnoughItems-forge-12.0.674.jar** and **RoughlyEnoughItems-fabric-12.0.674.jar**, I would easily tell I downloaded it wrong.
